tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,49 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers:
+#   proxy-httpd
+#   proxy-salt-broker
+#   proxy-squid
+#   proxy-ssh
+#   proxy-tftpd
+#
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_var('CONTAINER_IMAGE_TO_TEST');
+
+    assert_script_run("podman pull $image");
+    assert_script_run("podman run --name suma_image -dt $image");
+
+    record_info('Kernel', 'Test that kernel files are present');
+    validate_script_output("podman exec suma_image /bin/sh -c 'ls /boot'", sub { /initrd/ });
```

Comment:
This was a specific check for the Rancher image, you don't need this here, actually I think this will fail.